superset/datasets/api.py

```diff
@@ -162,7 +163,18 @@ class DatasetRestApi(BaseSupersetModelRestApi):
         "columns.type",
         "columns.uuid",
         "columns.verbose_name",
-        "metrics",
+        "metrics",  # TODO(john-bodley): Deprecate in 3.0.
```

Comment:
Ensures backwards compatibility, i.e., non-breaking change.

Reply:
what's the harm of keeping them both?